source/MaterialXRuntime/Private/PvtApi.h

```diff
@@ -96,11 +101,11 @@ class PvtApi
         {
             throw ExceptionRuntimeError("Given prim '" + prim.getName().str() + "' is not a valid nodedef");
         }
-        if (hasNodeDef(prim.getName()))
+        if (_nodedefs.count(prim.getName()))
         {
-            throw ExceptionRuntimeError("A nodedef with name '" + prim.getName().str() + "' is already registered");
+            _nodedefs.remove(prim.getName());
```

Comment:
Do we want to have the "last in" win's policy ?

Reply:
Yes. That's the current policy.